chatgpt_wrapper/gpt_shell.py

```diff
@@ -406,10 +407,36 @@ def do_ask(self, line):
         "`!ask` asks a question to chatgpt. It is purely optional. Example: `!ask what is 6+6` is the same as `what is 6+6`"
         return self.default(line)
 
+    def replace_file_strings(self, string):
+        # Regular expression to find all occurrences of !file and their arguments
+        pattern = r"!file\s+(\S+)"
+
+        # Search for all matches of the regular expression in the string
+        matches = re.findall(pattern, string)
+
+        # Loop through each match and replace the corresponding substring
+        for match in matches:
+            print("handle file" + match)
```

Comment:
We've been using Markdown list items for these kinds of status/progress messages, can you convert this one and the others below?